File: code/python/day-6/palindrome_number.py
'''program to find a number is palindrome or not'''
number=int(input("enter a number"))
temp=number
rem=0
c=0
while  number>0 :
    rem = int(number%10)
    number=int(number/10)  
    c =int((c*10)+rem)
    
        
if ( c==temp):
    
        print("number",temp,"is palindrome number")
        
else:
        
        print("number",temp,"is not palindrome")
    

File: code/c/day-11/palindrome_number.c
#include<stdio.h>
#include<conio.h>
void main()
{
int n,p,dub=0,rem;
printf("enter a number");
scanf("%d",&n);
p=n;
while(n!=0)
{
rem=n%10;
dub=dub+rem*10;
n=n/10;
}
if(sum==n)
printf("%d is a palindrome number");
else
printf("%d is not a palindrome number");
}
